packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -581,9 +608,11 @@ export class AtomicSearchBox {
         id={id}
         key={item.key}
         part={this.makeSuggestionPart(isSelected, isDivider, hasQuery)}
-        class={`flex px-4 h-10 items-center text-neutral-dark hover:bg-neutral-light cursor-pointer ${
+        class={`flex px-4 min-h-[40px] items-center text-neutral-dark hover:bg-neutral-light cursor-pointer ${
           isSelected ? 'bg-neutral-light' : ''
         }`}
+        // TODO: ideally would replace the below with the min-h-[40px] class above
```

Comment:
@ThibodeauJF here

Reply:
<img width="194" alt="Screen Shot 2022-06-08 at 11 08 07 AM" src="https://user-images.githubusercontent.com/4923043/172651798-1f7cc9d7-1193-40df-9421-449a7fd9c931.png">
works for me 🤔

Reply:
this is madness

Reply:
I am baffled, it works for me now too... 🤬